docs/administrators/operational_procedures.rst

```diff
@@ -33,23 +33,31 @@ Context
 Procedure
 ++++++++++
 
-1. Verify in `project.yml` that `install_mode` is set to `release`.
-2. Freeze all modules with
+1. Verify in ``project.yml`` that ``install_mode`` is set to ``release``.
```

Comment:
I would split this up: this part is relevant if you're using v1 modules. If you're using v2 you want to ensure that `pip.pre` is not set. Both if you're using both.